plugins/woocommerce/client/admin/config/core.json

```diff
@@ -38,7 +38,6 @@
 		"async-product-editor-category-field": false,
 		"launch-your-store": true,
 		"product-editor-template-system": false,
-		"blueprint": false,
```

Comment:
@chihsuan  1 more thing to confirm, previously blueprint is disabled by default in production build, but now I assume it's not?

Reply:
Yes, I think this is what we want to do according to the issue description.  @adrianduffell To confirm, do we still want to turn it on by default? As you just mentioned we may push back the blueprints MVP date.

Reply:
@chihsuan It's looking likely that we will push the release date back, I'll know later today. If needed, You could merge it with it disabled by default for now.

Reply:
Okay, I'll update the PR to disable it by default. Thanks for confirming!

Reply:
Updated in 645467c4fbb04f6371f94a21632152d4b01b2cab.